### ISSUE TITLE
Train PANet on my own dataset

### ISSUE THREAD

**Issue by karry-code** [kaixin96/PANet#40]
Hi there,
I'm working on IDD dataset for performing semantic segmentation on it. I find PANet is a quite good model to carry out few shot learning for semantic segmentation and is suitable for my task. It would be really helpful if you could guide on how can I train PANet on my dataset (IDD dataset). 

Thanks.

**Comment by kaixin96** [comment]
Hi @karry-code , the data loading code is quite VOC/COCO specific. If you want to use an external dataset, I suggest you checking out [torchmeta](https://github.com/tristandeleu/pytorch-meta). It has a cleaner abstraction for dataset class tailored for few-shot tasks. Thank you.

**Comment by karry-code** [comment]
Hi @kaixin96 , thanks a lot.

**Comment by roxhannes** [comment]
Hi @kaixin96 and @karry-code , 
I also would like to use PANet to segment some images of electrodes I am using for my research. I already have labeled images, but now I am struggeling how to train an own model. Can you help me how exactly it is possible to load external, non-published data, e.g. using torchmeta? 
Thanks and bests,
Hannes

**Comment by Meoooww** [comment]
Did you successfully apply it?

**Comment by roxhannes** [comment]
No not yet, if you have any useful approach, I am happy to hear about it...

**Comment by Patrick-sudo-cell** [comment]
@Meoooww Yes, I've applied it on my own dataset! The model shows a good performance, but i wanna say it is just an old model that cannot beat some recent models!
